cockatrice/src/settingscache.h

```diff
@@ -133,12 +142,18 @@ class SettingsCache : public QObject {
     QString getThemeName() const { return themeName; }
     QString getChatMentionColor() const { return chatMentionColor; }
     QString getChatHighlightColor() const { return chatHighlightColor; }
+    QString getSpoilerSavePath() const { return msSpoilerSavePath; }
     bool getPicDownload() const { return picDownload; }
     bool getNotificationsEnabled() const { return notificationsEnabled; }
+    bool getDownloadSpoilersStatus() const { return mbDownloadSpoilers; }
     bool getSpectatorNotificationsEnabled() const { return spectatorNotificationsEnabled; }
     bool getNotifyAboutUpdates() const { return notifyAboutUpdates; }
     ReleaseChannel * getUpdateReleaseChannel() const { return releaseChannels.at(updateReleaseChannel); }
     QList<ReleaseChannel*> getUpdateReleaseChannels() const { return releaseChannels; }
+    int getDownloadSpoilerTimeIndex() const { return mnDownloadSpoilersTimeIndex; }
+    float getDownloadSpoilerTimeMinutes() const { return msDownloadSpoilersTimeMinutes; }
```

Comment:
Why do you save in the settings file both the minutes and the entry index in the combobox? You should be able to derive one value from the other.
The problem is that if we ever ned to change the values on the list and keep them ordered, the mapping between indexes and minutes would change and this could cause odd effects when loading old settings in a new client version.